src/object_accessor.hpp

```diff
@@ -36,64 +36,8 @@
 #include <string>
 
 namespace realm {
-//
-// Value converters - template specializations must be implemented for each platform in order to call templated methods on Object
-//
-template<typename ValueType, typename ContextType>
-class NativeAccessor {
```

Comment:
Any ideas on how to do that? I guess I could just add a comment pointing at CppContext as a reference implementation of it. The previous approach of having a declaration that was never actually used had the problem of that there was nothing verifying that it was correct or complete (and it wasn't).